docs/src/user-guide/core-clp-s.md

```diff
@@ -12,11 +12,27 @@ Usage:
 ```
 
 * `archives-dir` is the directory that archives should be written to.
-* `input-path` is any new-line-delimited JSON (ndjson) log file or directory containing such files.
-* `options` allow you to specify things like which field should be considered as the log event's
-  timestamp (`--timestamp-key <field-path>`), or whether to fully parse array entries and encode 
-  them into dedicated columns (`--structurize-arrays`).
-  * For a complete list, run `./clp-s c --help`
+* `input-path` is any new-line-delimited JSON (ndjson) log file, KV-IR file, URL pointing to such
```

Comment:
The author of the ndjson spec has expressed willingness to deprecate the spec: https://www.github.com/ndjson/ndjson-spec/issues/35

Instead, JSON Lines (JSONL) was recommended. Shall we rename the references to JSONL?